scripts/uosc.lua

```diff
@@ -4487,6 +4492,26 @@ mp.add_key_binding(nil, 'chapters', create_self_updating_menu_opener({
 	end,
 	on_select = function(time) mp.commandv('seek', tostring(time), 'absolute') end,
 }))
+mp.add_key_binding(nil, 'editions', create_self_updating_menu_opener({
+	title = 'Editions',
+	type = 'editions',
+	list_prop = 'edition-list',
+	list_serializer = function(_, editions)
+		local items = {}
+		local editions = mp.get_property_native('edition-list', {})
```

Comment:
`editions` are already passed as a function parameter, no need to retrieve them again here.

Reply:
In fact, in my test. if we don't do this, it will get a nil value when switching edition and crash the script, just like this:
https://github.com/tomasklaen/uosc/blob/f9c062f9927985ec911fd97a8bb30987e5a53b69/scripts/uosc.lua#L4450-L4454